src/integrations/retry.ts

```diff
@@ -0,0 +1,84 @@
+import retry, { OperationOptions } from 'retry';
```

Comment:
I see you based it on a earlier version, did we not want to stick to the latest one?

Reply:
This is the latest one actually: https://github.com/tim-kos/node-retry/blob/master/package.json#L7

Reply:
Ah I compared to this? https://github.com/sindresorhus/p-retry/blob/main/index.js

Reply:
Yes that's what I thought :)
So we're all clear I guess